config_src/drivers/nuopc_cap/mom_ocean_model_nuopc.F90

```diff
@@ -733,6 +752,11 @@ subroutine ocean_model_restart(OS, timestamp, restartname, num_rest_files)
         endif
      endif
   endif
+  if (present(restartname)) then
```

Comment:
Thanks for the updates. I believe this line should be changed as follows:

```if (present(stoch_restartname)) then```

Reply:
You are correct. @jiandewang can you fix this?

Reply:
> 

done